project/src/utils.ts

```diff
@@ -20,4 +21,39 @@ export const getRatingStyle = (rating: number): number  => {
   }
 };
 
-export const getOffersInCity = (offers : Offer[], cityValue:string) => offers.filter(({city}) => (city.name === cityValue));
+export const isCheckedAuth = (authorizationStatus: string): boolean => authorizationStatus === AuthorizationStatus.Unknown;
+
+export const getOffersInCity = (offers: Offer[], cityValue: string): Offer[] => offers.filter(({ city }) => (city.name === cityValue));
+
+export const sortOffersRating = (offerA: Offer, offerB: Offer): number => {
+  if (offerA.rating < offerB.rating) {return 1;}
+  else if (offerA.rating > offerB.rating) {return -1;}
+  else {return 0;}
+};
```

Comment:
Так, пиши нормально. 
```suggestion
export const sortOffersRating = (offerA: Offer, offerB: Offer): number => {
  if (offerA.rating < offerB.rating) {
  return 1;
  } else if (offerA.rating > offerB.rating) {
  return -1;
  }  else {
  return 0;
  }
};
```

Нельзя так писать. 

ну а вообще, посмотри на объект Math и метод sign он очень хорошо подходит для этой цели